lib/ansible/modules/files/xml.py

```diff
@@ -800,7 +808,7 @@ def main():
 
     # Try to parse in the target XML file
     try:
-        parser = etree.XMLParser(remove_blank_text=pretty_print)
+        parser = etree.XMLParser(remove_blank_text=pretty_print,strip_cdata=strip_cdata_tags)
```

Comment:
https://app.shippable.com/github/ansible/ansible/runs/73860/1/tests
'''
 failure: The test `ansible-test sanity --test pep8` failed with 1 error:
lib/ansible/modules/files/xml.py:811:64: E231 missing whitespace after ','
'''